src/views/SignIn.vue

```diff
@@ -0,0 +1,111 @@
+<template>
+    <div class="row">
+  <div class="signIn w-100">
+    <form>
+      <b-col>
+        <b-form-group id="input-group-first-name" label="Your Firstname:" label-for="input-first-name">
```

Comment:
```suggestion
        <b-form-group id="input-group-first-name" label="First name" label-for="input-first-name">
```
https://medium.com/by-digiti/designing-the-perfect-input-field-aefb59bdd085